src/components/Board/Board.js

```diff
@@ -17,7 +18,10 @@ const Board = () => {
     getBoards().then((boardResponse) => {
       setBoard(boardResponse);
       setError(false);
-    }).catch(() => {
+    }).catch((err) => {
+      if (err.status === 401) {
```

Comment:
This is dead code, `err.status` is not defined. This statement is never `true`. Passing the response object to `new Error` doesn't seem to work like that.